processing/src/main/java/io/druid/segment/IndexIO.java

```diff
@@ -86,6 +86,7 @@
   public static final byte V8_VERSION = 0x8;
   public static final byte V9_VERSION = 0x9;
   public static final int CURRENT_VERSION_ID = V9_VERSION;
+  public static BitmapSerdeFactory LEGACY_FACTORY = new BitmapSerde.LegacyBitmapSerdeFactory();
```

Comment:
This means the only way to code the presence bitmap is concise, need to add this to docs thought

Reply:
this is only used in legacyIndexLoader and older columnserde that do not support nulls.